src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -306,6 +311,11 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             }
         });
 
+        // Hook up the hyperlinks from SettingContainer
+        _RegisterNavigationHandlers(GeneralStack());
+        _RegisterNavigationHandlers(AppearanceStack());
+        _RegisterNavigationHandlers(AdvancedStack());
```

Comment:
I'm not happy about this one. But we could leverage this to access the setting containers via code now.